homeworks/homework-1/app/src/components/Course.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from "react";
+import {Button, Col, ControlLabel, Form, FormControl, FormGroup} from "react-bootstrap";
+import Header from "./Header"
+
+class Course extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      greeting: "Edit course"
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <Header greeting={this.state.greeting}/>
+        <Form horizontal>
+          <FormGroup controlId="title">
+            <Col componentClass={ControlLabel} sm={4}>
+              Title
+            </Col>
+            <Col sm={2}>
+              <FormControl type="text" placeholder="Title"/>
```

Comment:
If you are writing a component in the JSX would be better all of props to separate per new line which you are providing to the component. That is a way of the clean code in React.

```
<Component
   prop={value}
   anotherProp={anotherValue}
/>
```